biggraphite/drivers/elasticsearch.py

```diff
@@ -424,7 +424,30 @@ def get_metric(self, metric_name, touch=False):
         """See the real Accessor for a description."""
         super(_ElasticSearchAccessor, self).get_metric(metric_name, touch=touch)
```

Comment:
Add a `TODO: ` to handle `touch=True` correctly (see cassandra.py, it's the part that will update `updated_on`)

Reply:
Done